packages/docs/docs/api.mdx

```diff
@@ -82,3 +82,48 @@ the DApp will also switch to that network. Support for specific DApp networks wi
   file="../../sdk/src/Fuel.test.ts"
   testCase="User getProvider on fuels-ts Wallet"
 />
+
+## Events
+
+The Fuel client emits events when certain actions occur. These events can be listened to by using the `on` method.
+
+### Usage
+
+The `fuel` object has an `events` property which is an enum of all the events that can be listened to. <br />
+The `on` method takes two arguments, the event name and a callback function. The callback function is passed the data associated with the event.
+
+### Connection
+
+Listen for a new connection to a node. Callback function receives the `connection` object.
+
+<CodeImport
+  file="../src/hooks/useIsConnected.tsx"
+  lineStart="10"
+  lineEnd="28"
+/>
+
+### Accounts
+
+Listen for when the client connects his accounts. Callback function receives the `account` object.
+
+<CodeImport
+  file="../examples/events/Accounts.tsx"
+  lineStart="24"
+  lineEnd="33"
+/>
+
+### Current Account
+
+Listen for when the current account is changed in the Fuel Wallet Extension. Callback function receives `account` address.
```

Comment:
```suggestion
This event is triggered when the current account is changed in the Fuel Wallet Extension. The callback function receives the `account` address.

This event is only triggered if the application has access to the current account address. Otherwise, it is not triggered.
```